src/common/components/body-class-modifier.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { DocumentManipulator } from 'common/document-manipulator';
+import * as React from 'react';
+
+export type BodyClassModifierProps = {
+    classNames: string[];
+    documentManipulator: DocumentManipulator;
+};
+
+// We user our own BodyClassModifier rather than using Helmet's similar functionality because we
+// need multiple BodyClassModifiers to be able to co-exist without overwriting each other or the
+// body classes office fabric maintains.
+export class BodyClassModifier extends React.Component<BodyClassModifierProps> {
+    public render(): JSX.Element {
+        return null;
```

Comment:
_"funny"_ comment: we went from a component that listen to the store for updates and render, calling the dependencies but not passing any of the new state, which was awkward, to a component that does pass params to it's dependencies on updates but that renders null every time :P Don't get me wrong, this looks way better than my original attempt, but it's still a tad odd that the component renders null no matter what.

Reply:
I agree that it feels a little odd; it's oddness we already had anyway (just indirectly via Helmet, before), but it's definitely a little strange-feeling for a react component to be side-effect-only